pkg/mkimage-raw-efi/install

```diff
@@ -245,8 +252,21 @@ if [ "$INSTALL_DEV" != "$INSTALL_PERSIST" ]; then
            fi
         fi
      done
-     if [ $DISKS_TO_MERGE_COUNT -eq 0 ]; then
-          # in case of only comma provided
+
+     if [ "$SKIP_ZFS_CHECKS" = false ]; then
```

Comment:
A bit unnecessay to run this code unless INSTALL_ZFS is also true.

Reply:
INSTALL_ZFS is always true here, line 233 sets that.

Reply:
Sorry misread the indentation